integration/spark/integrations/container/pysparkHiveCompleteEvent.json

```diff
@@ -26,198 +26,16 @@
                 }
             },
             "outputFacets": {
-                "outputStatistics": {
-                    "rowCount": 3,
-                    "size": 12
-                }
+                "outputStatistics": {}
```

Comment:
My changes doesn't serialize statistics. It has only zero
![Screenshot from 2021-09-17 11-04-41](https://user-images.githubusercontent.com/19191993/133756490-3d75df18-b9fc-41f3-9142-aa2494f1f51c.png)
s

Reply:
I've made changes to use provided jackson, but it cause to retrive empty statistics on Complete Event
The test is `SparkContainerIntegrationTest.testPysparkSQLHiveTest()`
Main branch retrives statistics, If I rollback `build.gradle` changes it also works.
jackson-databind is provided: `[Loaded com.fasterxml.jackson.databind.ser.impl.ReadOnlyClassToSerializerMap from file:/opt/miniconda3/lib/python3.7/site-packages/pyspark/jars/jackson-databind-2.6.7.1.jar]`
`[Loaded com.fasterxml.jackson.core.Versioned from file:/opt/miniconda3/lib/python3.7/site-packages/pyspark/jars/jackson-core-2.6.7.jar]`
Also tried to remove: `jackson-datatype-jsr310 `from our build - result is same, no statistics.

Reply:
@collado-mike, maybe you can suggest, what could go wrong?

Reply:
Is the `LogicalPlan` a `DataWritingCommand` at this step? Is that a difference between Spark 2 and 3?

Reply:
The problem occurs **on the same spark version** (2.4.1) and yes, it's `DataWritingCommand`